src/cljs/token/wallets/page.cljs

```diff
@@ -6,27 +6,30 @@
             [re-frame.core :as re-frame]
             [token.db :as db]))
 
-(defn wallet-uri [wallet-id]
-  [:a.wallet-btn {:href (str "#/wallet/" wallet-id)} "Open wallet"])
+(defn allTransactions []
```

Comment:
Lets use `all-transactions` here. Refer to https://github.com/bbatsov/clojure-style-guide#lisp-case.